R/tar_flood_exposure.R

```diff
@@ -17,20 +17,21 @@ zonal_pop_exposure <- function(floodscan_path=fp_fs,
                                flood_frac_thresh=0.005,
                                binarize_floodscan = T,
                                adm = lgdf_adm$adm0,
+                               country_seasons = c("MAM", "OND", "Annual"),
                                cols_keep = c("adm1_en", "adm1_pcode","adm0_en")
 ){
   tnc_fs <- tidync(floodscan_path)
   tnc_fs_filt <- fs_filter_bounds(fs_obj = tnc_fs,geometry = adm)
   r_fs <- fs_to_raster(fs_obj = tnc_fs_filt,band = "SFED_AREA")
```

Comment:
i've improved this code here: 
https://github.com/OCHA-DAP/ds-floodscan-ingest/blob/migrate-to-blob/R/fs_load_utils_tidync.R

Sometimes depending on the boundaries the raster would previously get a bit stretched